tests/v2/validation/charts/gatekeeper.go

```diff
@@ -18,97 +15,40 @@ const (
 	gatekeeperProjectName = "gatekeeper-project"
 	// namespace that is created without a label
 	RancherDisallowedNamespace = "no-label"
+	ConstraintType             = "constraints.gatekeeper.sh.k8srequiredlabels"
```

Comment:
Just to unify the steve naming, `ResourceSteveType` might distinguish GVK and Steve's differences more